ruleset/testing/tests/nextcloud.ini

```diff
@@ -1,48 +1,59 @@
+;  Copyright (C) 2015-2021, Wazuh Inc.
+;
+;  Tests for products:
+;    NextCloud.
```

Comment:
Do you have more information of Nexcloud version?

Reply:
It's a mistery. Not a nextcloud user